src/features/singleRight/offered/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from 'react-i18next';
+import * as React from 'react';
+import { PersonCheckmarkIcon } from '@navikt/aksel-icons';
+
+import { Page, PageHeader, PageContent, PageSize, PageContainer } from '@/components';
+import { useMediaQuery } from '@/resources/hooks';
+
+export const OverviewPage = () => {
+  const { t } = useTranslation('common');
```

Comment:
I get an assigned but never used warning on this one.

Reply:
I'll remove it